pkg/cio/io.go

```diff
@@ -254,12 +254,22 @@ func LogURI(uri *url.URL) Creator {
 
 // TerminalLogURI provides the raw logging URI
 // as well as sets the terminal option to true.
-func TerminalLogURI(uri *url.URL) Creator {
+func TerminalLogURI(uri *url.URL, setStderr ...bool) Creator {
```

Comment:
Why is `setStderr` variadic?  I only see index 0 being accessed.

Reply:
Responed to similar [comment](https://github.com/containerd/containerd/pull/10412#discussion_r1663531978)